### ISSUE TITLE
Decide on Color Scheme

### ISSUE THREAD

**Issue by vontell** [GoodEnoughSoftware/Mission-Complete#9]
Decide on the color scheme for the app as a whole. Alternatively, follow the path of Textra and allow it to be customized by the user. However, to start, we should at least have some sort of basic color scheme for simplicity of implementation.

**Comment by tpatel1781** [comment]
A couple options to consider; Todoist red color scheme, or maybe a monochrome theme, or a theme that goes from red to yellow to green as you complete an increasingly higher percentage of that day's tasks.

**Comment by vontell** [comment]
I like the idea of changing colors for the different tasks. If that is the case, we will need to decide if the toolbar is the same color as one of the cards, or is a completely separate color.

**Comment by tpatel1781** [comment]
Yeah that is true. What do you think of the monochrome theme. I think that would look clean, and it would be easy to rapidly prototype the apps functionality
